inngest/README.md

```diff
@@ -0,0 +1,23 @@
+# Inngest
+
+## Overview
+
+Send key metrics about your Inngest functions directly to your Datadog account.
+
+## Setup
+
+Install this integration by clicking "Connect Accounts" or "Install Integration" above. You may be asked to sign into your Inngest account, and then you'll be asked to authorize Inngest to access your Datadog organization.
```

Comment:
Since this content appears on a product docs page as well as an in-app integration tile, directions like "above" don't apply to all contexts.
```suggestion
Install this integration by clicking **Connect Accounts** or **Install Integration**. You may be asked to sign into your Inngest account and authorize Inngest to access your Datadog organization.
```